packages/firestore/exp/package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "@firebase/firestore/exp",
   "description": "A tree-shakeable version of the Firestore SDK",
-  "main": "../dist/exp/index.node.umd.js",
+  "main": "../dist/exp/index.node.esm2017.js",
```

Comment:
Should we update these names to match the ones you defined in firestore-exp? Alternatively, can we just drop this?

Reply:
I need to revert this. What do you mean `the ones you defined in firestore-exp`?

Reply:
We should use the same names in packages-exp/firestore/package.json and packages/firestore/exp/package.json

But it seems like we could just remove ` packages/firestore/exp/package.json`

Reply:
There is no `packages-exp/firestore`. Do you mean `packages-exp/firebase-exp/firestore/package.json`?
I need `packages/firestore/exp/package.json` for the location of the binary files for exp, so I can use them to update the fields in `packages/firestore/package.json`.

The exp release process is hacky and might look confusing. Hopefully we just need it for a short period of time.